Imagine/Filter/PostProcessor/MozJpegPostProcessor.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Liip\ImagineBundle\Imagine\Filter\PostProcessor;
+
+use Liip\ImagineBundle\Binary\BinaryInterface;
+use Liip\ImagineBundle\Model\Binary;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\ProcessBuilder;
+
+/**
+ * mozjpeg post-processor, for noticably better jpeg compression.
+ *
+ * @see http://calendar.perfplanet.com/2014/mozjpeg-3-0/
+ * @see https://mozjpeg.codelove.de/binaries.html
+ *
+ * @author Alex Wilson <a@ax.gy>
+ */
+class MozJpegPostProcessor implements PostProcessorInterface
+{
+    /** @var string Path to the mozjpeg cjpeg binary */
+    protected $mozjpegBin;
+
+    /** @var null|int Quality factor */
+    protected $quality;
+
+    /**
+     * Constructor.
+     *
+     * @param string   $mozjpegBin Path to the mozjpeg cjpeg binary
+     * @param int|null $quality    Quality factor
+     */
+    public function __construct(
+        $mozjpegBin = '/opt/mozjpeg/bin/cjpeg',
```

Comment:
This is already configurable here as a parameter - the value in the constructor shouldn't ever need to be used. https://github.com/antoligy/LiipImagineBundle/blob/implement-pngquant-mozjpeg/Resources/config/imagine.xml#L79